flang/lib/Optimizer/Transforms/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ add_flang_library(FIRTransforms
   VScaleAttr.cpp
   FunctionAttr.cpp
   DoConcurrentConversion.cpp
+  ../../Lower/OpenMP/Utils.cpp
```

Comment:
That's ugly indeed. I think we should separate everything under `flang/lib/Lower/OpenMP` to a separate target: `OpenMPLower`. But I think we should create a separate an upstream PR for this, WDYT?

Reply:
We have to do something like that, and I agree that it's something we'd want community comments for. The problem I see at the moment is that the only reason we want this right now is because of this pass, which we can't currently upstream. So, if we create an upstream PR to try to separate this, there won't be any follow-up patches to make use of it.

At this time, I would suggest to just leave it as you wrote it, adding a TODO comment right above this new line so we don't forget this is a workaround. And then, once we find the need to get something upstream that would benefit from sharing some code with lowering, we should try to do it right and remove this in the following merge from main. I hope that makes sense.